pkg/api/v20210131preview/openshiftcluster_validatestatic_test.go

```diff
@@ -78,6 +78,7 @@ func validOpenShiftCluster() *OpenShiftCluster {
 			NetworkProfile: NetworkProfile{
 				PodCIDR:     "10.128.0.0/14",
 				ServiceCIDR: "172.30.0.0/16",
+				SDNProvider: "OVNKubernetes",
```

Comment:
Use the constant here.

Reply:
https://github.com/Azure/ARO-RP/pull/1571/commits/7cf26fefa84635caced03ddb15651f0635027f4f#diff-d2a80e56830699a55c16cd21471ad6f85b8061d822f37b247f4b5021f337dfe0R81 

Constant